### PR TITLE
[#165715405] make ScreenHeaders look like the same

### PR DESCRIPTION
**Pull request by lmorelli986**
This PR aims to make headers be smaller and follow the same building structure.

This PR includes also the aspect of the Top Header layout as shown in the mockup https://projects.invisionapp.com/share/SXRS1MUVKGP#/screens

screen

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
## Affected stories

  * 🌟 [#165715405](https://www.pivotaltracker.com/story/show/165715405): Come CIT voglio vedere il componente ScreenHeader coerente su tutti gli screen

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by matteodesanti** [comment]
Please @lmorelli986 check margins and padding, as represented in the invision screen. Please see the attached image
image

**Comment by lmorelli986** [comment]
@matteodesanti thanks for documentation and feedback. Do you mean that Title should be spaced from the subtitle below by 16px? Because at the moment the spacing is between subT and content, not betw Title and subT.

**Comment by lmorelli986** [comment]
Added spacing above the title and below the subtitle.

Schermata 2019-05-21 alle 16 53 53

---


Simulator Screen Shot - iPhone 8 - 2019-05-21 at 16 52 06

File: ts/components/AbortOnboardingModal.tsx [hunk 1]
```diff
@@ -22,7 +23,7 @@ class AbortOnboardingModal extends React.PureComponent<Props> {
       <Modal visible={true} onRequestClose={this.props.onClose}>
         <Container>
           <AppHeader noLeft={true}>
-            <Right>
+            <Right style={styles.rightOnly}>
```
Comment (fpersico) [inline ts/components/AbortOnboardingModal.tsx]: The name "rightOnly" suggest this style is used when only the "Right" component is rendered but from the code is not the case.

File: ts/components/screens/BaseScreenComponent.tsx [hunk 2]
```diff
@@ -3,6 +3,7 @@ import { connectStyle } from "native-base-shoutem-theme";
 import mapPropsToStyleNames from "native-base/src/utils/mapPropsToStyleNames";
 import * as React from "react";
 
+import { ComponentProps } from "../../types/react";
```
Comment (fpersico) [inline ts/components/screens/BaseScreenComponent.tsx]: ComponentProps is already included in react-native. Can you try to use the one from react-native?

File: ts/components/ui/AppHeader.tsx [hunk 3]
```diff
@@ -1,10 +1,26 @@
 import { Header, NativeBase } from "native-base";
 import * as React from "react";
+import { StyleSheet } from "react-native";
 import variables from "../../theme/variables";
 import ConnectionBar from "../ConnectionBar";
 
 type Props = NativeBase.Header;
 
+const styles = StyleSheet.create({
+  header: {
```
Comment (fpersico) [inline ts/components/ui/AppHeader.tsx]: Can't we set the style to the header theme directly?

File: ts/components/screens/BaseScreenComponent.tsx (hunk 2 above)
Comment (lmorelli986) [reply ts/components/screens/BaseScreenComponent.tsx]: Fixed in c2c58540

File: ts/components/ui/AppHeader.tsx (hunk 3 above)
Comment (lmorelli986) [reply ts/components/ui/AppHeader.tsx]: fixed

**Pull request merged**